doc/sphinx/start_install.rst

```diff
@@ -1,217 +1,227 @@
+You will need to download the source code, digested observational data, and sample model data (:numref:`ref-download`). Afterwards, we describe how to install software dependencies using the `conda <https://docs.conda.io/en/latest/>`__ package manager (:numref:`ref-conda-install`) and run the framework on sample model data (:numref:`ref-configure` and :numref:`ref-execute`).
 
-**Summary of steps for installing the framework**
-
-You will need to download a) the source code, b) digested observational data, and c) two sets of sample model data (:numref:`ref-download`). Afterwards, we describe how to install necessary Conda environments and languages (:numref:`ref-install`) and run the framework on the default test case (:numref:`ref-configure` and :numref:`ref-execute`).
-
-You will need to download the source code, digested observational data, and sample model data (:numref:`ref-download`). Afterwards, we describe how to install software dependencies using the `conda <https://docs.conda.io/en/latest/>`__ package manager (:numref:`ref-install`, :numref:`ref-conda-env-install`) and run the framework on sample model data (:numref:`ref-configure` and :numref:`ref-execute`).
+Throughout this document, :console:`%` indicates the shell prompt and is followed by commands to be executed in a terminal in ``fixed-width font``. Variable values are denoted by angle brackets, e.g. <*HOME*> is the path to your home directory returned by running :console:`% echo $HOME`. 
 
 .. _ref-download:
 
 Obtaining the code
 ------------------
 
-The official repo for the MDTF code is hosted at the GFDL `GitHub account <https://github.com/NOAA-GFDL/MDTF-diagnostics>`__. We recommend that end users download and test the `latest official release <https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0-beta.3>`__.
+The official repo for the package's code is hosted at the NOAA-GFDL `GitHub account <https://github.com/NOAA-GFDL/MDTF-diagnostics>`__. We recommend that end users download the `latest official release <https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0-beta.3>`__.
 
-To install the MDTF framework, create a directory named ``mdtf`` and unzip the code downloaded from the `release page <https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0-beta.3>`__ there. This will create a directory titled ``MDTF-diagnostics-3.0-beta.3`` containing the files listed on the GitHub page. Below we refer to this MDTF-diagnostics directory as ``$CODE_ROOT``. It contains the following subdirectories:
+To install the MDTF package, create a directory named ``mdtf`` and unzip the code downloaded from the `release page <https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0-beta.3>`__ there. This will create a directory titled ``MDTF-diagnostics-3.0-beta.3`` containing the files listed on the GitHub page. Below we refer to the path to this MDTF-diagnostics directory as <*CODE_ROOT*>. It contains the following subdirectories:
```

Comment:
It would be less confusing, and consistent with the updated Readme.md file, to instruct all users to clone the repository into mdtf, then check out the tag for the latest release if they are mainly interested in running the PODs:
```
git clone https://github.com/NOAA-GFDL/MDTF-diagnostics
git checkout tags/v3.0-beta.3
```

Reply:
Thanks for catching this, @wrongkindofdoctor -- I forgot that the README was updated, because we haven't pushed the changes in `develop` to `main` yet (which we should do soon-ish). I'll include the git instructions as written on the README.

Unfortunately, I have some Strongly Held Opinions™ on what the README should look like, but I'll put those into a separate issue and PR rather than holding up progress on this one.